[2024-11-18T16:40:11.800+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T16:40:11.876+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.test scheduled__2024-11-18T16:30:00+00:00 [queued]>
[2024-11-18T16:40:11.920+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.test scheduled__2024-11-18T16:30:00+00:00 [queued]>
[2024-11-18T16:40:11.922+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T16:40:12.030+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): test> on 2024-11-18 16:30:00+00:00
[2024-11-18T16:40:12.043+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1901) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T16:40:12.047+0000] {standard_task_runner.py:72} INFO - Started process 1903 to run task
[2024-11-18T16:40:12.049+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'test', 'scheduled__2024-11-18T16:30:00+00:00', '--job-id', '384', '--raw', '--subdir', 'DAGS_FOLDER/fetch_data.py', '--cfg-path', '/tmp/tmpvyuzow3v']
[2024-11-18T16:40:12.051+0000] {standard_task_runner.py:105} INFO - Job 384: Subtask test
[2024-11-18T16:40:12.177+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.test scheduled__2024-11-18T16:30:00+00:00 [running]> on host c06adc74012b
[2024-11-18T16:40:12.644+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='test' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T16:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T16:30:00+00:00'
[2024-11-18T16:40:12.646+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T16:40:12.647+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T16:40:12.648+0000] {logging_mixin.py:190} INFO - Current task name:test state:running start_date:2024-11-18 16:40:11.879193+00:00
[2024-11-18T16:40:12.649+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T16:40:12.651+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T16:40:12.669+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2024-11-18T16:40:12.696+0000] {logging_mixin.py:190} INFO - PostgreSQL connection successful!
[2024-11-18T16:40:12.698+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T16:40:12.725+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T16:40:12.726+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=test, run_id=scheduled__2024-11-18T16:30:00+00:00, execution_date=20241118T163000, start_date=20241118T164011, end_date=20241118T164012
[2024-11-18T16:40:12.745+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T16:40:12.747+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T16:40:12.748+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 16:40:00.084917+00:00
[2024-11-18T16:40:12.749+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T16:40:12.787+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T16:40:12.841+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T16:40:12.848+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
